README.md

```diff
@@ -0,0 +1,61 @@
+# AMQP client
+
+This crate provides a simple AMQP client capable of processing messages of a
+defined type. The client provides an unbounded channel to interact with the
+client, either to submit new messages to the queue or to receive messages from
+the queue.
+
+### Usage
+
+```rust
```

Comment:
If this snippet compiles, we can add something to `lib.rs` so that it compiles this. https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#include-items-only-when-collecting-doctests

```rust
#[doc = include_str!("../README.md")]
#[cfg(doctest)]
pub struct ReadmeDoctests;
```

Reply:
Interesting, I didn't know about this. It took me a little bit but I got it working.

Reply:
Awesome!